server/src/main/java/org/opensearch/index/mapper/WildcardFieldMapper.java

```diff
@@ -678,7 +679,7 @@ public Query termsQuery(List<?> values, QueryShardContext context) {
             StringBuilder pattern = new StringBuilder();
             for (Object value : values) {
                 String stringVal = BytesRefs.toString(value);
-                builder.add(matchAllTermsQuery(name(), getRequiredNGrams(stringVal)), BooleanClause.Occur.SHOULD);
+                builder.add(matchAllTermsQuery(name(), getRequiredNGrams(stringVal), false), BooleanClause.Occur.SHOULD);
```

Comment:
@HUSTERGS Are these lines covered by the tests above?
If not, should we add more tests for coverage?

Reply:
I think it is covered by the YAML test introduced in #15936 and #15607.
As far as I know, `case_insensitive` parameter is not supported for terms query. If you apply this parameter to terms query, you will get an error like this:
```
{
  "error": {
    "root_cause": [
      {
        "type": "parsing_exception",
        "reason": "[terms] query does not support [case_insensetive]",
        "line": 5,
        "col": 27
      }
    ],
    "type": "parsing_exception",
    "reason": "[terms] query does not support [case_insensetive]",
    "line": 5,
    "col": 27
  },
  "status": 400
}
```

Plus, the YAML test put two docs with content of `ABCD` and `AbCd`, and the terms query YAML test checked the response only contains the case **sensitive** one.